hub/subscribe.go

```diff
@@ -87,25 +104,21 @@ func (h *Hub) initSubscription(w http.ResponseWriter, r *http.Request) (*Subscri
 	authorizedAlltargets, authorizedTargets := authorizedTargets(claims, false)
 	subscriber := NewSubscriber(authorizedAlltargets, authorizedTargets, topics, rawTopics, templateTopics, retrieveLastEventID(r))
 
-	if subscriber.LastEventID != "" {
-		h.sendMissedEvents(w, r, subscriber)
+	pipe, err := h.transport.CreatePipe(subscriber.LastEventID)
+	if err != nil {
```

Comment:
This condition is not tested, it can be tested by using a test transport that returns an error